contact_calc/vanderwaals.py

```diff
@@ -78,6 +81,10 @@ def compute_vanderwaals(traj_frag_molid, frame, index_to_atom, sele1, sele2, geo
         if atom1.chain == atom2.chain and abs(atom1.resid - atom2.resid) < res_diff:
             continue
 
+        #Check and continue if disulphide bond
+        if atom1.resid in disulfide_cys and atom2.resid in disulfide_cys:
```

Comment:
It's a little far-fetched, but you could imagine two independent disulphide bridges: A-B and C-D where A and C have vdw interactions. Those would be discarded with this logic, I think.